pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -216,6 +216,32 @@ public void TestDdaSearchCometAutoTolerance()
             RunFunctionalTest();
             Assert.IsFalse(IsRecordMode);
         }
+        [TestMethod, NoParallelTesting(TestExclusionReason.RESOURCE_INTENSIVE), NoUnicodeTesting(TestExclusionReason.TIDE_UNICODE_ISSUES)]
+        public void TestDdaSearchTide()
+        {
+            TestFilesZip = @"TestFunctional\DdaSearchTest.zip";
+
+            if (RedownloadTools)
+                foreach (var requiredFile in CometSearchEngine.FilesToDownload)
+                    if (requiredFile.Unzip)
+                        DirectoryEx.SafeDelete(requiredFile.InstallPath);
+                    else
+                        FileEx.SafeDelete(Path.Combine(requiredFile.InstallPath, requiredFile.Filename));
+
+            TestSettings = new DdaTestSettings
+            {
+                SearchEngine = SearchSettingsControl.SearchEngine.Tide,
+                FragmentIons = "b,y",
+                Ms2Analyzer = "Default",
+                PrecursorTolerance = new MzTolerance(15, MzTolerance.Units.ppm),
+                FragmentTolerance = new MzTolerance(1.0005),
+                AdditionalSettings = new Dictionary<string, string>(),
+                ExpectedResultsFinal = new ExpectedResults(123, 297, 358, 1074, 144)

```

Comment:
You will probably need to change these numbers in order for this test to pass.
If you modify the code so that the property "IsRecordMode" (on line 317) is true, then the test will write out the expected numbers instead of failing, and you can copy those numbers from the test output and put them here and then change "IsRecordMode" back to false.